cypress/integration/collections_spec.js

```diff
@@ -71,36 +78,35 @@ describe('Dashboard Collections Page', () => {
     it('collection page has button to edit the collection', () => {
       const name = 'MOD09GQ';
       const version = '006';
+
       cy.visit(`/#/collections/collection/${name}/${version}`);
       cy.contains('a', 'Edit').should('exist').as('editCollection');
       cy.get('@editCollection')
         .should('have.attr', 'href')
         .and('include', `#/collections/edit/${name}/${version}`);
       cy.get('@editCollection').click();
 
-      cy.contains('.heading--large', `Edit ${name}___${version}`);
-
-      // There are issues updating the react form which uses ace editor.
-      // Neither cy .clear nor ace editor works.
-      // As a workaround, we add additional parameter at the end of collection metadata
-      const meta = '"meta": "testmetadata"}';
-      cy.get('textarea').type(`{backspace},${meta}`, { force: true });
+      cy.contains('.heading--large', `Edit ${name}___${version}`).should('exist');
 
+      // update collection and submit
+      const meta = 'metadata';
+      cy.editJsonTextarea({ data: { meta }, update: true });
```

Comment:
Can you also update an existing field, such as set `duplicateHandling` to a different value?

Reply:
Yeah, if you pass in this `update: true` parameter, then whatever you specify for `data` will be merged with the existing object using `_.assign` from Lodash: https://lodash.com/docs/4.17.10#assign